package.json

```diff
@@ -27,9 +28,11 @@
     "lint": "eslint modules"
   },
   "dependencies": {
+    "@types/history": "^2.0.41",
+    "@types/react": "^0.14.52",
```

Comment:
We shouldn't introduce new dependencies that don't affect the majority of users. If this can be added another way, that would be better (I'm not familiar with TS tooling)